cogs/events.py

```diff
@@ -1,27 +1,89 @@
 import traceback
+import textwrap
+from .utils import constants
 
 import discord
 from discord.ext import commands
 
+PREFIX = constants.COMMAND_PREFIX
+
 
 class Custom(commands.Cog):
 
     def __init__(self, client):
         self.client = client
+        self.dev_mode = constants.DEV_MODE
 
     @commands.Cog.listener()
     async def on_message(self, message):
         if message.author.id == self.client.user.id:
             return
 
-        if message.content.lower().startswith("help"):
-            await message.channel.send("Hey! Why don't you run the help command with `>>help`")
+        if message.author.bot:  # make sure the bot doesn't respond to other bots
+            return
```

Comment:
```py
       if message.author.id == self.client.user.id:
            return
```
This is checking if the author is the bot itself and returning earlier if ture.
```py
        if message.author.bot:  # make sure the bot doesn't respond to other bots
            return
```
This is doing the same thing, but this one is checking for all bots, not only self.

That means there is no point to keep both conditions, the second one covers the first one and also checks for other bots, meaning that one is enough, the first condition can be removed.